saleor/graphql/checkout/mutations/order_create_from_checkout.py

```diff
@@ -51,7 +51,8 @@ class Arguments:
 
     class Meta:
         description = (
-            f"{ADDED_IN_32} Create new order from existing checkout. {PREVIEW_FEATURE}"
+            f"{ADDED_IN_32} Create new order from existing checkout. Can be used only"
+            f" by apps. {PREVIEW_FEATURE}"
```

Comment:
We could also do something like this:

```
class Meta:
    auto_permission_message = False
    description = (
        f"{ADDED_IN_32} Create new order from existing checkout. Requires the "
        f"following permissions: AUTHENTICATED_APP and HANDLE_CHECKOUTS."
        f"{PREVIEW_FEATURE}"
    )
```

This way we override the autogenerated message about required permissions, and instead of writing our own description we use AUTHENTICATED_APP to represent that it requires an authenticated app.

Reply:
Thanks <3